src/curl.cpp

```diff
@@ -2369,9 +2410,26 @@ void S3fsCurl::insertV2Headers()
   }
 }
 
+void S3fsCurl::insertIBMHeaders()
+{
+  requestHeaders = curl_slist_sort_insert(requestHeaders, "Authorization", ("Bearer " + S3fsCurl::AWSAccessToken).c_str());
+
+  if(op == "PUT" && path == mount_prefix + "/"){
+    // create bucket request
```

Comment:
Can you expand this comment a bit since I don't understand why you add this extra header.

Reply:
Will change to:
// ibm-service-instance-id header is required for bucket creation requests